src/portal-components/DetailsPage.tsx

```diff
@@ -448,7 +457,7 @@ export const DetailsPageSynapseConfigArray: React.FC<{
     <>
       {synapseConfigArray.map((el: RowSynapseConfig, index) => {
         const id = COMPONENT_ID_PREFIX + index
-        const { standalone, resolveSynId, showTitleSeperator = true } = el
+        const { resolveSynId, showTitleSeperator = true } = el
```

Comment:
Passes entire el (RowSynapseConfig) to getSynapseComponent(), so no need to pull out the standalone value